src/BenchmarkDotNet.Diagnostics.Windows/PmcDiagnoser.cs

```diff
@@ -231,6 +234,9 @@ public MisspredictRateColumn(Dictionary<Benchmark, PmcStats> results)
             public bool AlwaysShow => false;
             public ColumnCategory Category => ColumnCategory.Diagnoser;
             public int PriorityInCategory => 1;
+            public QuantityType QuantityType => QuantityType.None;
+            public string GetName(ISummaryStyle style) => ColumnName;
```

Comment:
should the GetName implementation handle `style.PrintUnitsInContent` to include `%` sign?